dotcom-rendering/src/components/SecureSignup.importable.tsx

```diff
@@ -237,6 +260,9 @@ export const SecureReCAPTCHASignup = ({
 
 	useEffect(() => {
 		setCaptchaSiteKey(window.guardian.config.page.googleRecaptchaSiteKey);
+		void resolveEmailIfSignedIn().then((email) => {
+			setSignedInUserEmail(email);
+		});
```

Comment:
Not a blocker, but maybe simpler?

```suggestion
		void resolveEmailIfSignedIn().then(setSignedInUserEmail);
```